hoomd/write/gsd.py

```diff
@@ -90,43 +100,36 @@ class GSD(Writer):
         * constraints/
         * pairs/
 
-
-    .. seealso::
-
+    See Also:
         See the `GSD documentation <http://gsd.readthedocs.io/>`_ and `GitHub
         project <https://github.com/glotzerlab/gsd>`_ for more information on
         GSD files.
 
-    .. note::
-
-        When you use *filter_* to select a subset of the whole system,
-        :py:class:`GSD` will write out all of the selected particles in
-        ascending tag order and will **not** write out **topology**.
-
-    .. note::
+    Note:
+        When you use ``filter`` to select a subset of the whole system, `GSD`
+        will write out all of the selected particles in ascending tag order and
+        will **not** write out **topology**.
 
+    Tip:
         All logged data chunks must be present in the first frame in the gsd
         file to provide the default value. Some (or all) chunks may be omitted
         on later frames.
```

Comment:
I had a question about this -- how do you write a quantity to only frame 0? Like `type_shapes` for a system with fixed shape parameters.

Reply:
Good point. You ensure that the logger is present on the first frame written. Namely, don't create the writer, write a few frames, and then set the log attribute.

Reply:
@joaander I am having trouble understanding what you mean in the second half of your comment. Do you just mean that it will automatically skip writing `type_shapes` on later frames if the data is identical to that of frame 0? If that's not what you mean, can you provide an example?

Reply:
The writer will not automatically skip writing log entries. They have no defaults. What I am trying to say is do this:
```
gsd = hoomd.write.GSD(...)
gsd.log = log
sim.run(1e6)
```

Not this:
```
gsd = hoomd.write.GSD(...)
sim.run(1e6)
gsd.log = log
sim.run(1e6)
```

And not this:
```
gsd = hoomd.write.GSD(...)
gsd.log = log
sim.run(1e6)
log.add(something)
sim.run(1e6)
```

In the second and third cases the writer will write log entries to later frames in the file that are not present in the earlier frames. Logged entries have no defaults, so readers cannot guess at what they might be in earlier frames.